templates/articles/record_article_page.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base_page.html' %}
+
+{% load static %}
+{% load wagtailcore_tags %}
+{% load sections_tags %}
+{% load wagtailmetadata_tags %}
```

Comment:
Nitpick: Could we load these in-line instead? or atleast group them a little, it's very verbose.

Reply:
This template is here to be replaced anyway... so I would say it's not worth the effort